src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -76,6 +77,33 @@ void setUserStore(UserStore userStore) {
     this.userStore = userStore;
   }
 
+  public static String replaceUrls(String text){
```

Comment:
Would be cool to see some tests for this function :cat:

Reply:
on it!